test/user.test.js

```diff
@@ -294,17 +293,15 @@ describe('User', function() {
         },
         function(next) {
           User.login({email: 'b@c.com', password: 'bar'}, function(err, accessToken) {
-            accessTokenId = accessToken.userId;
             if (err) return next(err);
-            assert(accessTokenId);
+            assert(accessToken.userId);
```

Comment:
While you are changing this code, can we improve the assertion to something like the following? (Use `expect` instead of `assert`, verify the value is actually the expected user id, not just any truthy value.)

```js
expect(accessToken.toJSON()).to.have.property('userId', userIds[0]);
```

Same comment applies to the next check below.